### PR TITLE
Unown outposts with Chaos Wave bombardement weapon

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt [hunk 1]
```diff
@@ -36,6 +36,33 @@ Part
...
             ]
 
+        EffectsGroup    // players can order chaos wave used on enemies outposts
+            scope = And [
+                Planet
+                InSystem id = Source.SystemID
+                OrderedBombardedBy condition = Source
+                Or [
+                    OwnedBy affiliation = EnemyOf empire = Source.Owner
+                    Unowned
```
Comment (geoffthemedio) [inline default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt:45]: if it's already unowned, the message about losing ownership is misleading...

File: default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt [hunk 2]
```diff
@@ -62,8 +89,36 @@ Part
...
+                Population high = 0
+            ]
+            activation = And [
+                Unowned
+                InSystem
+            ]
+            priority = [[POPULATION_DEFAULT_PRIORITY]]
+            effects = [
+                SetOwner empire = [[UNOWNED_EMPIRE_ID]]
+                RemoveSpecial name = "GAIA_SPECIAL"
```
Comment (geoffthemedio) [inline default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt]: should maybe also mention that in the sitrep?

File: default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt (hunk 1 above)
Comment (o01eg) [reply default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt:45]: Changed it.

File: default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt (hunk 2 above)
Comment (o01eg) [reply default/scripting/ship_parts/Bombard/SP_CHAOS_WAVE.focs.txt]: Added separate message.

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -7768,6 +7768,12 @@ Derelict Fuel
 EFFECT_BIOWEAPON
 %planet% has lost %rawtext% population due to a %shippart% attack.
 
+EFFECT_BIOWEAPON_GAIA
+%planet% has lost %rawtext% population and [[special GAIA_SPECIAL]] due to a %shippart% attack.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: shouldn't say it lost Gaia unless it had it

Comment (o01eg) [reply default/stringtables/en.txt]: @geoffthemedio  It looks like I should use separate effect group or `If` with condition on Gaia special. Before it wasn't mentioned.

Comment (geoffthemedio) [reply default/stringtables/en.txt]: > separate effect group

sounds reasonable...

Comment (o01eg) [reply default/stringtables/en.txt]: @geoffthemedio Splitted removing Gaia special.

**Comment by geoffthemedio** [comment]
didn't test, but looks OK

**Pull request merged**